gcp/table_gcp_compute_instance.go

```diff
@@ -293,6 +293,7 @@ func listComputeInstances(ctx context.Context, d *plugin.QueryData, h *plugin.Hy
 	}
 
 	filterQuals := []filterQualMap{
+		{"name", "name", "string"},
```

Comment:
In this table, can you please add `name` as an optional key column to the `ListConfig`? When querying the `gcp_compute_instance` table, if someone specifies a `name`, then the `GetConfig` would be used, so this change wouldn't make any functional difference, but I think it would help make the table definition more consistent when people see the `name` column here in the `filterQualMap`.

Reply:
I agree; I've done this in both the `gcp_compute_disk` and `gcp_compute_instance` tables.